### DISCUSSION TITLE
Adding newlines to Common Lisp docstrings

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1572] [Q&A] [answered]
Is it possible to add newlines to Common Lisp docstrings? I mean newlines like the Emacs Lisp docstring convention:

```lisp
(defun square (x)
  "Compute the square.
Multiplies the argument X by itself and returns the result.
No input validation is done, so the function assumes X to be
a number."
  (* x x))
```

In SEdit, if I type `ENTER` in the docstring I get `↑M` and typing `Ctrl-J` inserts `↑J`. Escaping with `%` doesn't help.

**Comment by masinter** [comment, marked as answer]
A control-M doesn't display prettily in SEdit, apparently, but it is the proper encoding of a newline in Lisp. 
If you type an 'enter' in SEdit it will create the correct string, which shows up on the file as a bare eol with no left-margin indentation. If you save square on a file and 'see squarefile' or just 'pp square' it will show up
![image](https://github.com/Interlisp/medley/assets/1116587/9dbb29b5-4189-4105-b6b6-4b67df1110b7)
